src/l500/ac-trigger.h

```diff
@@ -62,6 +62,11 @@ namespace ivcam2 {
         double _last_temp = 0;
         std::shared_ptr< temp_check > _temp_check;
 
+        class rgb_stability;
+        bool _need_to_wait_for_color_sensor_stability = false;
```

Comment:
Shouldn't it be atomic?

Reply:
If it does need to be, then we have other variables that need to be. After looking at the code, I think it's fairly safe and that the only sets are at the end-points (trigger/stop) and from set_color_frame. The one complication would be stop & set_color_frame happening together -- but they both set it to false, so I don't see how they'd conflict.